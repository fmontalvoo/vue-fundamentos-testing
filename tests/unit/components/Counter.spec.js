import { shallowMount, mount } from '@vue/test-utils';

import Counter from '@/components/Counter.vue';

describe('Counter.vue', () => {
    test('Debe hacer match con el snapshot', () => {

        const wrapper = shallowMount(Counter);

        expect(wrapper.html()).toMatchSnapshot();
    });
});