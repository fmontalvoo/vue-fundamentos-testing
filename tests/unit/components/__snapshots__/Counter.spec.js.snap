// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter.vue Debe hacer match con el snapshot 1`] = `
<h2>Counter: 1<sup>2</sup> = 1</h2>
<div><button class="minus">-</button><button class="plus">+</button></div>
`;
